packages/blade/src/utils/getStringChildren/getStringChildren.ts

```diff
@@ -0,0 +1,17 @@
+import type { StringChildrenType } from '~src/_helpers/types';
+
+const getStringFromReactText = (
+  childReactText: StringChildrenType | undefined,
+): string | undefined => {
+  if (!childReactText) {
```

Comment:
Let's check for actual undefined. 

Or else, This will fail
```
<BadgeComponent>{0}</BadgeComponent>
```

Reply:
Oh right. good catch 👯

Reply:
done